main.c

```diff
@@ -53,21 +53,47 @@
 #include "nrf_delay.h"
 #include "boards.h"
 
+#define ID 6579
```

Comment:
It's better to use more unique names than `ID`, `SIZE` and `TIME`. Otherwise, you can meet a conflict with some other macro/variables. For example: `DEVICE_ID`, `DEVICE_ID_SIZE`, `BLINK_TIME`